src/components/Correction/index.ts

```diff
@@ -0,0 +1 @@
+export * from './IncorporationAgreement'
```

Comment:
FWIW, I went the other way here and got rid of most index.ts for components. In Create UI this got rid of circular / nested imports and some weirdness.

But this change may be fine here. Up to you as things evolve how you want to go. (It's easy to change to one way or the other.)

Reply:
The reason I reorganized things in Create UI was to make it easy(er) to tell what components a filing type uses, and also what filing types a component is used by.

Reply:
Sounds good, will have a closer look at this! I see what you mean